bin/initialize_db.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.."
+mkdir -p data/postgres
+if test ! -d  data/postgres/base; then
+	pg_ctl initdb -D data/postgres
+	pg_ctl start -w -D data/postgres -o "-h localhost"
+	createdb voterunner
+	pg_ctl stop -D data/postgres
+fi
```

Comment:
I would like that - but that has to be coded first. Right now the app does it on demand, which is fine with me.